neuro_san/service/agent_server.py

```diff
@@ -186,4 +193,5 @@ def serve(self):
             concierge_service,
             server)
 
+        self.notify_started.set()
         self.server_lifetime.run()
```

Comment:
At this point, gRPC server internal state is initialized, and initial agents repository is constructed.
So tell (in this case, http server) it's OK to start running.